src/main/java/com/google/step/snippet/servlets/SearchServlet.java

```diff
@@ -136,7 +146,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     } catch (InterruptedException | RejectedExecutionException e) {
       allCards = Collections.emptyList();
     }
-
+    if (!preferredSite.isEmpty()) {
```

Comment:
Can we put a newline before this line for some visual separation between these steps?